ibm_cloud_sdk_core/__init__.py

```diff
@@ -12,11 +12,27 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Classes and helper functions used by generated SDKs.
+
+classes:
+    BaseService: Abstract class for common functionality between each service.
+    DetailedResponse: The object returned from successful service operations.
+    IAMTokenManager: Requests and refreshes IAM tokens using an apikey, and optionally a client_id and client_secret pair.
+    JWTTokenManager: Abstract class for common functionality between each JWT token manager.
+    CP4DTokenManager: Requests and refreshes CP4D tokens given a username and password.
+    ApiException: Custom exception class for errors returned from service operations.
+
+functions:
+    datetime_to_string: Serializes a datetime to a string.
+    string_to_datetime: De-serializes a string to a datetime.
+    get_authenticator_from_environment: Get authenticator from external sources.
+    read_external_sources: Get config object from external sources.
+"""
 
 from .base_service import BaseService
 from .detailed_response import DetailedResponse
 from .iam_token_manager import IAMTokenManager
 from .jwt_token_manager import JWTTokenManager
 from .cp4d_token_manager import CP4DTokenManager
 from .api_exception import ApiException
-from .utils import datetime_to_string, string_to_datetime, get_authenticator_from_environment, read_external_sources
+from .utils import datetime_to_string, string_to_datetime, read_external_sources
```

Comment:
Is there a reason to remove `get_authenticator_from_environment`? It is used in the watson SDKs: https://github.com/watson-developer-cloud/python-sdk/blob/master/ibm_watson/assistant_v1.py#L29

Reply:
@ehdsouza I added it back to https://github.com/IBM/python-sdk-core/pull/35/files#diff-273d7ea91f33cc339a1dcb6d2568b5adR44. I moved it because there were circular dependencies that were interfering with proper type hinting.

Reply:
But there is now a dissonance here between the comment listing the classes and functions exported (I love the comment!  Thanks for adding that) and the actual exports in this file.  So perhaps you could add a comment here to explain with that one method isn't exported here, and where it is exported from instead.

Reply:
I've fixed the comment.